code/modules/mob/living/carbon/human/human.dm

```diff
@@ -295,15 +295,6 @@
 	var/obj/item/organ/external/affecting = get_organ(ran_zone(dam_zone))
 	apply_damage(5, BRUTE, affecting, run_armor_check(affecting, "melee"))
 
-/mob/living/carbon/human/bullet_act()
```

Comment:
This one was called after the other version of the proc. I kept the order the same